remote-api/src/main/java/org/alfresco/rest/api/actions/ActionValidator.java

```diff
@@ -26,13 +26,39 @@
 
 package org.alfresco.rest.api.actions;
 
+import java.util.List;
+
 import org.alfresco.rest.api.model.rules.Action;
 import org.alfresco.service.Experimental;
 
 @Experimental
 public interface ActionValidator
 {
+
+    String ALL_ACTIONS = "all";
+
+    /**
+     * Provides validation logic for given action.
+     */
     void validate(Action action);
 
+    /**
+     * Indicates whether validation is enabled. Could be based on property value in a specific implementation.
+     */
     boolean isEnabled();
+
+    /**
+     * Returns priority of validator (applied to bulk validation in @see {@link org.alfresco.rest.api.impl.mapper.rules.RestRuleActionModelMapper})
+     * @return priority expressed as int
```

Comment:
Maybe you could mention whether a higher or lower int has more priority?

Reply:
I will